modules/face/src/facemarkAAM.cpp

```diff
@@ -560,8 +560,8 @@ namespace face {
         reduce(Ys,sumYs, 0, CV_REDUCE_SUM);
 
         //calculate the normrnd
-        double normX = sqrt(sumXs.at<float>(0)+sumXs.at<float>(1));
-        double normY = sqrt(sumYs.at<float>(0)+sumYs.at<float>(1));
+        double normX = sqrt(Mat(sumXs.reshape(1)).at<float>(0)+Mat(sumXs.reshape(1)).at<float>(1));
```

Comment:
@alalek the error in build debug is fixed here, but why it is not checked anymore?

Reply:
This build is optional, it is not triggered automatically.
Debug build is passed, thanks!